melodic/distribution.yaml

```diff
@@ -4106,6 +4106,18 @@ repositories:
         release: release/melodic/{package}/{version}
       url: https://github.com/ros-gbp/sbpl-release.git
       version: 1.3.1-0
+  schunk_modular_robotics:
+    doc:
+      type: git
+      url: https://github.com/ipa320/schunk_modular_robotics.git
+      version: indigo_release_candidate
```

Comment:
My guess is that you will want to use the `melodic_release_candidate` branch here